Bottom: 7226508d705d86bb7d601bab8dd5cb56ed817ead
Top:    a2d0eefb36f12cefa2c314bcbbe58bb6f6e83b22
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 16:31:27 +0000

Remove left over debug


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index ba5c857..3d525eb 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1503,8 +1503,6 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
 #endif
                               copper_appearance,
                               NULL));
-
-    //break;
   }
